src/themes/teams/components/Icon/svg/icons/add.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react'
+import { TeamsSvgIconSpec } from '../types'
+
+export default (
+  <svg viewBox="0 0 32 32" role="presentation" className="app-svg icons-add" focusable="false">
```

Comment:
The className should come through the classes property provided to the icon. @VyshnaviDasari  please have a look at the example from the call.tsx or format.tsx svgs.

Reply:
@alinais I changed the classNames . Can you check it.